python/mxnet/numpy/multiarray.py

```diff
@@ -1318,25 +1318,59 @@ def array(object, dtype=None, ctx=None):
 @set_module('mxnet.numpy')
 def zeros(shape, dtype=_np.float32, **kwargs):
     """Return a new array of given shape and type, filled with zeros.
-    This function currently only supports storing multi-dimensional data
-    in row-major (C-style).
 
     Parameters
     ----------
-    shape : int or tuple of int
-        The shape of the empty array.
-    dtype : str or numpy.dtype, optional
-        An optional value type (default is `numpy.float32`). Note that this
-        behavior is different from NumPy's `ones` function where `float64`
-        is the default value, because `float32` is considered as the default
-        data type in deep learning.
+    shape : int , tuple of ints or list of ints
+        Shape of the new array, e.g., ``(2, 3)`` or ``2``.
+    dtype : data-type, optional
+        The desired data-type for the array, e.g., `numpy.int8`.  Default is
+        `numpy.float32`.
     ctx : Context, optional
         An optional device context (default is the current default context).
 
     Returns
     -------
     out : ndarray
-        Array of zeros with the given shape, dtype, and ctx.
+        Array of zeros with the given shape, dtype, and order.
+
+    Notes
+    -----
+    - Not support ndarray type
+    - Not support custom dtype
```

Comment:
What does those two notes mean? Please explain a bit.

Reply:
if input is np.array((3,2)) , it will return error!

Reply:
resolve